src/type/AnyIdType.php

```diff
@@ -0,0 +1,24 @@
+<?php declare(strict_types=1);
+
+namespace hiqdev\php\billing\type;
+
+/**
+ * AnyIdType class.
+ *
+ * This class simplifies the creation of Type instances where the ID is always
+ *  set to `Type::ANY`, which is used to represent a wildcard or "any" match for the ID.
+ *  It reduces the repetitive creation of Type objects with `Type::ANY` as the ID.
+ *
+ *  Example:
+ *  Instead of:
+ *      new Type(Type::ANY, 'monthly,leasing');
+ *  You can use:
+ *      new AnyIdType('monthly,leasing');
+ */
+class AnyIdType extends Type
+{
+    public function __construct($name)
+    {
+        parent::__construct(self::ANY, $name);
+    }
+}
```

Comment:
This class acts as a factory around the parent object and does not bring additional behavior. If we need to create a subclass in the future, e.g. `RemoteType`, we will be forced to create `AnyIdRemoteType` as well.

I'd rather create a named constructor method in the Type class:

```php
Type::anyId('monthly,vps')
```

That's what named constructors are created for.

Reply:
created Type::anyIdI() method instead of AnyTypeId class